.github/workflows/checklist.yml

```diff
@@ -0,0 +1,29 @@
+name: Checking PR checklist
+permissions: read-all
+on:
+  pull_request:
+    types: [opened, edited, synchronize]
+
+jobs:
+  check-pr-description:
+    runs-on: ubuntu-latest
+    steps:
+      - id: task-completed
+        uses: chromaui/task-completed-checker-action@main
+        with:
+          repo-token: '${{ secrets.GITHUB_TOKEN }}'
```

Comment:
Don't forget to add this secret!

Reply:
I believe it's created for me: https://docs.github.com/en/actions/security-for-github-actions/security-guides/automatic-token-authentication